### PR TITLE
Top Watched and Top Trending cards

### PR DESCRIPTION
**Pull request by enginnerd**

### CODE REVIEW

File: WealthNOW/app/src/main/java/com/finnerds/wealthnow/services/FeedService.java [hunk 1]
```diff
@@ -216,19 +287,94 @@ public void done(List<ParseStock> parseStocks, ParseException e) {
...
+                public void onFailure(String errorMessage) {
+//                    Toast.makeText(getActivity(), "error updating watchlist", Toast.LENGTH_SHORT).show();
+                }
+            });
+        } catch (Exception ex) {
+            Log.e("HomeFragment", "Error refreshing news data");
+        }
+    }
+
+    public static Comparator<Stock> PopularityDscComparator = new Comparator<Stock>() {
```
Comment (mikeng13) [inline WealthNOW/app/src/main/java/com/finnerds/wealthnow/services/FeedService.java]: We have a StockComparators class for this :)

File: WealthNOW/app/src/main/java/com/finnerds/wealthnow/viewholders/feeds/TopWatchedFeedItemViewHolder.java [hunk 2]
```diff
@@ -1,19 +1,218 @@
...
+            i++;
+            tvTicker_wi3.setText(stocks.get(i).symbol);
+            tvPrice_wi3.setText(String.valueOf(stocks.get(i).lastTradePrice));
+            tvChange_wi3.setText(String.valueOf(stocks.get(i).priceChange));
+            tvChange_wi3.setTextColor(context.getResources().getColor(getPriceColor(stocks.get(i))));
+            tvPopularity_wi3.setText(String.valueOf(stocks.get(i).popularity));
+            tvTickerFullName_wi3.setText(stocks.get(i).name);
+        }
+
+        llwatchItem1.setOnClickListener(new View.OnClickListener() {
```
Comment (mikeng13) [inline WealthNOW/app/src/main/java/com/finnerds/wealthnow/viewholders/feeds/TopWatchedFeedItemViewHolder.java]: This is just nitpicky but we can use ButterKnife's @OnClick annotation. Feel free to just leave as is and we can later refactor.

File: WealthNOW/app/src/main/res/layout/trending_feed_item.xml [hunk 3]
```diff
@@ -1,8 +1,269 @@
...
+        android:textAppearance="?textLarge"
+        android:textAllCaps="false"
+        android:text="Apple Inc."
+        android:singleLine="true"
+        android:ellipsize="end"
+        android:id="@+id/tvTickerFullName_wi3"
+        />
+    <!-- end listview -->
+
+    <!--<Button-->
```
Comment (mikeng13) [inline WealthNOW/app/src/main/res/layout/trending_feed_item.xml]: Get rid of this?

File: WealthNOW/app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -32,4 +32,5 @@
         <item>Change Asc</item>
         <item>Change Dsc</item>
     </string-array>
+    <string name="title_activity_discover">DiscoverActivity</string>
```
Comment (mikeng13) [inline WealthNOW/app/src/main/res/values/strings.xml]: Do we need a better name here?

**Comment by mikeng13** [comment]
Looks great! Just some dumb nitpick stuff...

**Comment by enginnerd** [comment]
Sounds good. I'll merge and we can refactor later.

**Pull request merged**